app/views/layouts/crawler.html.erb

```diff
@@ -23,9 +23,6 @@
     <%= render partial: "layouts/noscript_footer" %>
     <%= theme_lookup("footer") %>
     <%= theme_lookup("body_tag") %>
-    <% if show_browser_update? %>
-      <div class="buorg"><div><%= I18n.t("js.browser_update").html_safe %></div></div>
```

Comment:
IIRC this was at the bottom of the page to ensure that crawlers don't use the 'browser update' warning as a synopsis of the page. Any chance we can keep it down here in the HTML?

Reply:
Ahh ok that makes sense. The only issue is to make it appear at the top, we need to absolute position, which ends up cutting off the page title.